scripts/install_redis_cli.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
```

Comment:
2016 -> 2020

Can we have a python script instead of bash?

Reply:
Removed.